src/app/page.js

```diff
@@ -440,13 +448,14 @@ export default function Home() {
             </div>
           </div>
         </section>
-        <section
-          id="contact"
-          className="w-full py-12 md:py-24 lg:py-32 bg-muted"
-        >
-          <div className="container space-y-12 px-4 md:px-6">
-            <div className="flex flex-col items-center justify-center space-y-4 text-center">
-              <div className="space-y-2" />
+        <section id="faq" className="w-full md:py-24 lg:py-30 bg-muted">
```

Comment:
@bibschan Change the content a bit. I got some initial questions but you should review/complete the answers. 👍🏼 😘

Reply:
nicely done! I updated the copy just now, merging :) obrigada!